README.md

```diff
@@ -1 +1,77 @@
-# terraform-google-gcloud
\ No newline at end of file
+# terraform-google-gcloud
+
+This module allows you to use gcloud, gsutil, and any gcloud component in Terraform. Sometimes, there isn't Terraform GCP support for a particular feature, or you'd like to do something each time Terraform runs (ie: upload a file to a Kubernetes pod) that lacks Terraform support.
+
+This module *does not* create any resources on GCP itself, rather exposes the GCP SDK to you for usage in null resources & external data resources.
+
+## Usage
+
+Basic usage of this module is as follows:
+
+```hcl
+module "gcloud" {
+  source  = "terraform-google-modules/gcloud/google"
+  version = "~> 0.1"
+
+  platform = "linux"
+  additional_components = ["kubectl", "beta"]
+
+  create_command  = "${module.gcloud.gcloud} version"
```

Comment:
@milesmatthias Aren't we going to change this so you can just specify `create_command = "version"`

Reply:
It's already been updated, I just didn't update the README. Changing now...